docs/source/guides/worklock_guide.rst

```diff
@@ -7,29 +7,301 @@ WorkLock Guide
-Once the bidding period closes, the WorkLock contract doesn't accept more bids, although it still accepts cancellations during an additional time window.
-At the end of this cancellation period, stake-locked NU will be distributed according to the following principles:
-
- - Each bidder receives, at least, the minimum amount of NU needed to stake.
- - All bids will be greater or equal to the minimum allowed bid.
- - In addition to the minimum amount of NU, each bidder receives a portion of the remaining NU, distributed pro rata across all participants, taking into consideration their bid surplus with respect to the minimum bid.
- - If the resulting NU amount is above the maximum allowed NU to stake, then such a bidder has their bid partially refunded until the corresponding amount of NU is within the allowed limits.
+Once the contribution period closes, the WorkLock contract does not accept more bids, but it will still accept
+cancellations during an additional time window. At the end of this cancellation period, the claiming window opens and
+stake-locked NU token allocations can be claimed by participants. Stake-locked NU will be distributed according to
+the following principles:
+
+ - All of the tokens held by WorkLock will be distributed.
+ - All bids must be greater than or equal to the minimum allowed bid.
+ - For each bid, the surplus above the minimum allowed bid is called the `bonus`; all bids are composed of a `base` bid (fixed minimum bid) and a `bonus` bid (variable amount).
+ - Each bidder will receive at least the minimum amount of NU needed to stake.
+ - Once all bidders have been assigned the minimum amount of NU, each bidder with a `bonus` will receive a portion of the remaining NU, distributed pro rata across all participants, taking into consideration only their bonus amounts.
+ - If the resulting NU amount distributed to a bidder is above the maximum allowed NU to stake, then such a bidder has their bid partially refunded until the corresponding amount of NU is within the allowed limits.
 
 Finally, if WorkLock participants use that stake-locked NU to run a node, the NU will eventually unlock and their escrowed ETH will be returned in full.
 
 
+Hypothetical Bidding Scenarios
+------------------------------
+
+.. note::
+
+    To reduce complexity, calculations are performed in a step-wise manner and may lead to minor rounding differences
+    in the determined values.
+
+For each scenario, assume that:
+
+ #. WorkLock holds 280,000,000 NU tokens and the minimum bid is 15 ETH.
+ #. The minimum amount of NU required to stake is 15,000 NU and the maximum stake size is 4,000,000 NU.
+ #. The total number of bidders is 1000 bidders (including you) with a total of 50,000 ETH committed (including your bid).
+ #. For our purposes, a `whale` bid is a bid that causes the calculated stake size to be larger than the maximum stake size (4,000,000 NU).
+
+
+Scenario 1: Resulting stake size does not exceed maximum stake size
```

Comment:
Maybe add clarification in the name of the scenario that there are no whale bid at all?

Reply:
✔️